src/zcash/util.cpp

```diff
@@ -15,13 +15,12 @@ std::vector<unsigned char> convertIntToVectorLE(const uint64_t val_int) {
 // Convert bytes into boolean vector. (MSB to LSB)
 std::vector<bool> convertBytesVectorToVector(const std::vector<unsigned char>& bytes) {
     std::vector<bool> ret;
-    ret.resize(bytes.size() * 8);
+    ret.resize(bytes.size() * __CHAR_BIT__);
```

Comment:
It seems that `__CHAR_BIT__` is not part of the c++ standard.
I would suggest using `CHAR_BIT`, which is guaranteed by the standard to be defined in `climits`.

Reply:
done